app/1_Pre_EDA.py

```diff
@@ -4,7 +4,7 @@
 from module.EDA_BE import display_eda_info, round_decimal_columns, check_and_fill_null_values, check_and_remove_duplicates
 
 # Insert Tittle and markdown
-st.title("Hi this Fahmi Exploratory Data Analysis🕵️")
+st.title("Hi this Fahmi again Exploratory Data Analysis🕵️")

```

Comment:
alright great for testing , now i have put the ruleset into the main branch , i will accept for this time for testing , next , we will see more for improvement of the code

Reply:
done change